### PR TITLE
Send view events to the SDR metrics service when embedding

### PR DESCRIPTION
**Pull request by thatbudakguy**
Ref #1683

### CODE REVIEW

File: app/views/embed/_analytics.html.erb [hunk 1]
```diff
@@ -16,3 +16,20 @@
   }
   gtag('config', 'G-80J719XWE5', args)
 </script>
+<script src="https://unpkg.com/ahoy.js"></script>
```
Comment (jcoyne) [inline app/views/embed/_analytics.html.erb:19]: Can we pin to a specific version?  Should we add an integrity attribute https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity ?

Comment (thatbudakguy) [reply app/views/embed/_analytics.html.erb]: Both good suggestions! Added.

**Comment by thatbudakguy** [comment]
related config change: https://github.com/sul-dlss/shared_configs/pull/2107/files

File: config/settings.yml [hunk 2]
```diff
@@ -5,6 +5,7 @@ purl_conn_timeout: 2
 purl_feedback_email: 'some-email-address@distribution-list-host.stanford.edu'
 stacks_url: 'https://stacks.stanford.edu'
 iiif_info_url: 'https://library.stanford.edu/iiif/viewers'
+metrics_api_url: 'https://sdr-metrics-api-prod.stanford.edu'
```
Comment (jcoyne) [inline config/settings.yml:8]: Do we want to configure all of our instances to send the data to the prod metrics server?  It seems this even will send the data I generate by running it locally.

Comment (thatbudakguy) [reply config/settings.yml]: Updated so that it's not provided by default, and if there's no value set, we don't track views.

**Review by jcoyne — APPROVED** [review]

**Pull request merged**